asv.conf.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Can we move this inside the `benchmarks` folder? It'd be nice to keep the top-level directory as neat as possible.

Reply:
For example, [numpy](https://github.com/numpy/numpy/tree/main/benchmarks) has a top-level `benchmarks` dir that contains all the configs, with another `benchmarks` subdirectory where the asv benchmarks themselves are actually defined. I like this organization because it keeps all the benchmarking-related stuff self-contained (though maybe a better naming scheme would be `benchmarking` for the top-level directory and `benchmarks` for the subdirectory)